samples/event_manager_proxy/modules/sensor_simulated.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+
+#include "measurement_event.h"
+#include "control_event.h"
+#include "ack_event.h"
+#include "config_event.h"
+
+#define SENSOR_SIMULATED_THREAD_STACK_SIZE 800
+#define SENSOR_SIMULATED_THREAD_PRIORITY 1
+#define SENSOR_SIMULATED_THREAD_SLEEP 500
+#define MODULE sensor_sim
+
+static K_THREAD_STACK_DEFINE(sensor_simulated_thread_stack,
+			     SENSOR_SIMULATED_THREAD_STACK_SIZE);
+static struct k_thread sensor_simulated_thread;
+
+static int8_t value1;
```

Comment:
note that overflows on signed integers are undefined behavior

Reply:
direct copy from event_manager sample.